clients/vault/src/lib.rs

```diff
@@ -59,3 +59,17 @@ cfg_if::cfg_if! {
 		pub type DecimalsLookupImpl = primitives::AmplitudeDecimalsLookup;
 	}
 }
+
+pub fn tokio_spawn<F>(task_name: &str, future: F) -> tokio::task::JoinHandle<F::Output>
+	where
+		F: std::future::Future + Send + 'static,
+		F::Output: Send + 'static,
+{
+	cfg_if::cfg_if!{
+		if #[cfg(all(tokio_unstable, feature = "allow-debugger"))] {
```

Comment:
allow naming tasks, if `allow-debugger` feature is on.

Reply:
Do we need to add this check? Or can we just always use the builder pattern and in case the `allow-debugger` feature is not enabled, the name just wouldn't show?

Reply:
the [builder only works for tokio unstable](https://docs.rs/tokio/latest/tokio/task/join_set/struct.Builder.html), unfortunately.